src/custom-types.json

```diff
@@ -347,7 +347,11 @@
   "application/x-pkcs12": {
     "compressible": false
   },
-  "application/x-rar-compressed": {
+  "application/x-protobuf": {
+    "compressible": true,
+    "sources": ["https://developers.google.com/api-client-library/java/google-http-java-client/reference/1.20.0/com/google/api/client/protobuf/ProtocolBuffers"]
```

Comment:
Is there a source you can add about the compressibility of the type?

Reply:
There is lots of discussion on the [mapbox vector tile spec github issues page](https://github.com/mapbox/vector-tile-spec/issues/27) on compression, but a more authoritative source which clearly states that vector tiles (which are protocol buffers) can be compressed or uncompressed, is the GDAL documentation on [Mapbox Vector Tiles](https://gdal.org/drivers/vector/mvt.html).